### PR TITLE
Dict: Add more corrections

### PR DESCRIPTION
**Pull request by sebweb3r**

### CODE REVIEW

**Comment by sebweb3r** [comment]
I've spotted secant, which is a valid word in the code dictionary. Shall I remove it?

File: codespell_lib/data/dictionary_code.txt [hunk 1]
```diff
@@ -24,6 +25,7 @@ nto->not
 objext->object
 od->of
 process'->process's
+protecten->protection
 referer->referrer
 rela->real
 secant->second
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary_code.txt:31]: secant:
https://dictionary.cambridge.org/de/worterbuch/englisch/secant

Comment (larsoner) [reply codespell_lib/data/dictionary_code.txt:31]: Indeed `secant` is real. Maybe move to `rare`?

Comment (sebweb3r) [reply codespell_lib/data/dictionary_code.txt:31]: Rare is used in default. This would annoy much more people than code.

Comment (larsoner) [reply codespell_lib/data/dictionary_code.txt:31]: Let's just delete it for now then. It's not even that close a misspelling...

**Pull request merged**